src/components/Modal/Modal.vue

```diff
@@ -756,7 +756,7 @@ export default {
 
 	// Make modal full screen on mobile
 	@media only screen and (max-width: $breakpoint-mobile / 2) {
-		&:not(&--small) .modal-container {
+		.modal-container {
```

Comment:
Changing this media query to also addressing the small modal has the reason that I designed the small modal initially to be used e.g. for prompts with a limited height. In reality it gets used in many places mostly with mostly a pretty big height. Adjusting this makes sure that the mobile view is aligned to the other sizes and also looks better, imo.